src/app/homepage/pages/techniques/mvc/mvc.component.html

```diff
@@ -62,7 +62,7 @@ <h4>Fastify</h4>
     library. In order to create a MVC application with fastify, we have to install following packages:
   </p>
   <pre><code class="language-bash">
-$ npm i --save fastify point-of-view handlebars</code></pre>
+$ npm i --save fastify fastify-formbody fastify-static point-of-view handlebars</code></pre>
```

Comment:
Is this change really necessary? I mean, those packages are optional and may depend on the specific scenario of the application. E.g., if i am building a REST API only, i may not need the `fastify-formbody` or `fastify-static` packages.

I think, it would be better to just highlight those packages (or rather the overall `fastify` website) in a specific `hint` block?